lib/ltutils.coffee

```diff
@@ -23,7 +23,11 @@ class LTool
 # TODO add support for configurable extensions
 
 # In: current tex file; Out: file to be compiled
-module.exports.get_tex_root = (editor) ->
+module.exports.get_tex_root = (editor, project) ->
+
+  if project?.TEXroot?
```

Comment:
I think this check should be moved to the bottom of this function, i.e., so *if* the root isn't set in the current text-editor, then and only then do we check the project file. In my view, this results in the least surprise if someone uses both `%!TEX root =` comments in the file and a `TEXroot` setting in the project.